pkg/runtime/executor.go

```diff
@@ -131,3 +131,13 @@ func (e *Executor) Execute() (results Output, err error) {
 	// successful
 	return results, nil
 }
+
+// GetTotalPolicyCount will return the total count of policies in all engines
+func (e Executor) GetTotalPolicyCount() int {
```

Comment:
Absolutely, this should go into the policy package. There is some code that may help with this, which is a refactor of the policy engine code, but it has not been completed yet. I will try to get to this soon. But in the meantime, this code itself should be placed in the policy package itself.

Reply:
In the policy package, added a GetRuleCount() method on engine. In the runtime, the executor has the list of policy engines and it will be able to get the total rule count.